features/dump.feature

```diff
@@ -6,6 +6,10 @@ Feature: The dump command
     Then the output should contain "Util.system! dumple development"
       And the output should contain "Successfully dumped the development database"
 
+    When I run `geordi dump -d primary`
+    Then the output should contain "Util.system! dumple development"
+      And the output should contain "Successfully dumped the development database"
+
```

Comment:
Please also have this as a separate scenario.
- check for "Dumping the primary development database"
- check for `Util.system! dumple development primary`
- check for "Successfully dumped the primary development database"

Reply:
You actually did not do any changes here yet. Please either add support for dumping multiple local databases, or remove that test.